### PR TITLE
Support service defined using parameters inside of class

### PR DESCRIPTION
**Pull request by Oliboy50**
cc @twdmnd

### CODE REVIEW

File: src/Tests/Fixtures/Resources/config/minimal.xml [hunk 1]
```diff
@@ -20,9 +20,12 @@
...
 
-    <!-- Fake GuzzleHttp\Client service -->
+    <!-- Fake GuzzleHttp\Client services -->
+    <parameters>
+        <parameter key="part_of_guzzle_class_as_param">GuzzleHttp</parameter>
+    </parameters>
     <services>
         <service id="test.guzzle1" class="GuzzleHttp\Client"/>
-        <service id="test.guzzle2" class="GuzzleHttp\Client"/>
+        <service id="test.guzzle2" class="%part_of_guzzle_class_as_param%\Client"/>
```
Comment (Oliboy50) [inline src/Tests/Fixtures/Resources/config/minimal.xml:29]: this was not supported before, that's what this PR is for

**Comment by gmillet** [comment]
👍

**Comment by NastasiaSaby** [comment]
:+1:

**Comment by Oliboy50** [comment]
to v1.0.0

**Pull request merged**